crates/ruff_python_ast/generate.py

```diff
@@ -0,0 +1,581 @@
+#!/usr/bin/python
+
+from __future__ import annotations
+from dataclasses import dataclass
+from pathlib import Path
+import re
+from subprocess import check_output
+import tomllib
+from typing import Optional
```

Comment:
nit: could you run Ruff on this file with its isort rules enabled? (Sorry! 😄)

Reply:
Sure!  I was relying on however `ruff` is run via `uvx pre-commit run --all-files`.  Is this an option we want to turn on globally for the pre-commit hook?